docs/usage.rst

```diff
@@ -192,6 +195,16 @@ This is done on a per-course basis and can be modified in 3 different locations
     .. image:: images/site_configuration_data.png
 
 
+    .. note::
+        The site configuration module uses a cache to improve the read performance for succesive reads. Unlike other caches in the platform that use memcached, this is a local thread cache that is implemented in the django-crud package.
+
+        When you change values in the site configuration object, the easiest way to break the cache is perform a process restart.
```

Comment:
I think `performing` instead of `perform` would be better

Reply:
I made it even simpler. Simpler is better